Photino.HelloPhotino.React/Program.cs

```diff
@@ -6,13 +6,20 @@ namespace Photino.HelloPhotino.React;
 
 class Program
 {
+    public static bool IsDebugMode = true;
+
     [STAThread]
     static void Main(string[] args)
     {
+#if RELEASE
+        IsDebugMode = false;
+#endif
         PhotinoServer
             .CreateStaticFileServer(args, out string baseUrl)
             .RunAsync();
 
+        string appUrl = IsDebugMode ? "http://localhost:3000" : $"{baseUrl}/index.html";
```

Comment:
React runs on PORT 3000 by default, so this anticipates that feature. The developer is free to retrieve this from an environment variable.